torch/quantization/fx/quantize.py

```diff
@@ -350,6 +376,14 @@ def _convert(self, model, inplace=False, debug=False, is_dynamic_quant=False):
         self.modules = dict(model.named_modules())
 
         matches = self._find_matches(model.graph, self.modules, self.patterns)
```

Comment:
same here, would it make sense to make all the matching happen in `_find_matches`?